src/Domain/LeanCode.DomainModels/Ids/TypedIdConverter.cs

```diff
@@ -12,37 +16,143 @@ public class StringTypedIdConverter<TId> : JsonConverter<TId>
 
     public override void Write(Utf8JsonWriter writer, TId value, JsonSerializerOptions options) =>
         writer.WriteStringValue(value.Value);
+
+    public override TId ReadAsPropertyName(
+        ref Utf8JsonReader reader,
+        Type typeToConvert,
+        JsonSerializerOptions options
+    ) => Read(ref reader, typeToConvert, options);
+
+    public override void WriteAsPropertyName(Utf8JsonWriter writer, TId value, JsonSerializerOptions options) =>
+        writer.WritePropertyName(value.Value);
 }
 
 [System.ComponentModel.EditorBrowsable(System.ComponentModel.EditorBrowsableState.Never)]
 public class IntTypedIdConverter<TId> : JsonConverter<TId>
     where TId : struct, IRawTypedId<int, TId>
 {
+    private static readonly int SpanSize = int.MinValue.ToString(CultureInfo.InvariantCulture).Length;
```

Comment:
I'm not sure if we should deduce the span size like that. IMO this is no better than `const int SpanSize = 11` because it assumes that `int.MinValue.ToString(CultureInfo.InvariantCulture).Length > int.MaxValue.ToString(CultureInfo.InvariantCulture).Length`

Reply:
I would, but that is only an opinion, go with `SpanSize = 16` (or 32, doesn't really matter) and would add a comment explaining that.

Reply:
Doesn't matter to me, hardcoded this as you've suggested for now.